src/EditorFeatures/TestUtilities/Formatting/AbstractNewDocumentFormattingServiceTests.cs

```diff
@@ -40,6 +40,14 @@ internal Task TestAsync<T>(string testCode, string expected, (Option2<T>, T)[]?
                 parseOptions);
         }
 
+        internal Task TestAsync(string testCode, string expected, (OptionKey, object)[]? options = null, ParseOptions? parseOptions = null)
```

Comment:
> `(OptionKey, object)[]? options = null`

In the future, I recommend `OptionsCollection` for this. Very easy to initialize a new instance and set strongly-typed values. I'll look into updating this case after this PR merges.